aten/src/ATen/core/ivalue.h

```diff
@@ -351,6 +351,8 @@ struct CAFFE2_API IValue final {
   }
   std::string toNone() const {
     AT_ASSERT(isNone());
+    // TODO why are we returning a string for this...
```

Comment:
It is a holdover, introduced in 0ef5cfd28cd9c7be6cf6b3848882f548d55a8525, where there was a macro that defined the toString method as operator << on the toX method. Since there is no toNone(), one was added. This method should just be deleted because that macro no longer exists.